java/client/src/main/java/glide/api/commands/VectorSearchBaseCommands.java

```diff
@@ -0,0 +1,33 @@
+/** Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.commands;
+
+import glide.api.models.commands.vss.FTCreateOptions.FieldInfo;
+import glide.api.models.commands.vss.FTCreateOptions.IndexType;
+import java.util.concurrent.CompletableFuture;
+
+public interface VectorSearchBaseCommands {
+    // TODO GlideString???
```

Comment:
Is there a particular reason why this is just a TODO and not implemented here?

Reply:
We will implement it later if needed (is it needed?). I don't want to overload this PR by duplicated code and tests.